packages/rx_bloc_generator/lib/src/builders/state_method.dart

```diff
@@ -11,6 +11,7 @@ class _StateMethod implements _BuilderContract {
         (b) => b
           ..docs.addAll(['']) // A new line
           ..returns = refer(
+            // ignore: deprecated_member_use
```

Comment:
Why we ignore the warning instead of fixing it ?

Reply:
Well, this issue is only present in the pipeline and not locally (using the same flutter version!). The analysis warnings present `withNullability` option in `getDisplayString(withNullability: true)` as deprecated even though the official documentation mentions nothing and treats the method as valid:

![Screenshot 2024-05-31 at 13 16 36](https://github.com/Prime-Holding/rx_bloc/assets/61236867/6e95d94e-bf11-4826-b9f4-0b8a1094f5c8)

This ignores the pipeline warnings without any effect on the `rx_bloc_cli` package.

Reply:
Please check the flutter and packages versions are the same as on your local env

Reply:
Yep, just double checked but it seems that the flutter version, dart version as well as the packages are all the same (even when fetched using `flutter pub get` or `flutter pub upgrade`).